packages/edit-site/src/components/editor/index.js

```diff
@@ -142,35 +161,7 @@ export default function Editor( { listViewToggleElement, isLoading } ) {
 	const secondarySidebarLabel = isListViewOpen
 		? __( 'List View' )
 		: __( 'Block Library' );
-	const blockContext = useMemo( () => {
```

Comment:
All this logic has been moved directly to the `EditorProvider` component. It's going to benefit the post editor as well.